src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -17,6 +17,10 @@ export interface OrderData {
     approveUrl: string;
 }
 
+export interface OrderStatus {
+    status: 'APPROVED' | 'CREATED';
```

Comment:
Looks like it could be then 2 statuses...

```suggestion
    status: 'APPROVED' | 'CREATED' | string;
```

Reply:
fixed